sklearn/metrics/tests/test_ranking.py

```diff
@@ -716,6 +707,23 @@ def test_binary_clf_curve_implicit_pos_label(curve_func):
         np.testing.assert_allclose(int_curve_part, float_curve_part)
 
 
+@pytest.mark.parametrize("curve_func", [
+    precision_recall_curve,
+    roc_curve
+])
+@pytest.mark.parametrize("labels", [
+    np.array([b"a", b"b"], dtype='<S1'),
+    [b'a', b'b']
+])
+def test_binary_clf_curve_implicit_bytes_pos_label(curve_func, labels):
+    # Check that using bytes class labels raises an informative
+    # error for any supported string dtype:
```

Comment:
```suggestion
    # error for any supported string dtype:
    labels = _convert_container([b'a', b'b'], labels_type)
```

`_convert_container` should be imported from `sklearn.testing._convert_container`